pkg/landscaper/webhook/validator.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	lscore "github.com/gardener/landscaper/pkg/apis/core"
+	lsvalidation "github.com/gardener/landscaper/pkg/apis/core/validation"
+	"github.com/gardener/landscaper/pkg/utils/webhook"
+)
+
+type validator struct {
+	logger logr.Logger
+}
+
+// NewValidator creates a new Validator
+func NewValidator() webhook.Validator {
+	return &validator{
+		logger: log.Log.WithName("landscaper-validator"),
```

Comment:
```suggestion
func NewValidator(log logr.Logger) webhook.Validator {
	return &validator{
		logger: log.WithName("landscaper-validator"),
```

Reply:
Was using the wrong logger. It's still `logger: log.Logger.WithName(...)` though.